trulens_eval/trulens_eval/feedback/provider/cortex.py

```diff
@@ -101,7 +101,7 @@ def _create_chat_completion(
             kwargs['messages'] = messages
 
         elif prompt is not None:
-            kwargs['messages'] = [{"role": "system", "content": prompt}]
+            kwargs['messages'] = [{"role": "user", "content": prompt}]
```

Comment:
@sfc-gh-dhuang fyi this used to throw an error on my end that the last message must use the user role